jadx-gui/src/main/java/jadx/gui/ui/codearea/SearchBar.java

```diff
@@ -158,20 +158,20 @@ private void search(int direction) {
 			} catch (BadLocationException e) {
 				LOG.error("Caret move error", e);
 			}
-		}
+		}*/
```

Comment:
This removes those crutches) from changing the background in the input field.  In general, this needs to be rewritten so that it shows which position we are in from the found, like ide search bar.